README.md

```diff
@@ -36,7 +36,7 @@ Alby is open-source and currently in alpha stage. Our goal is to create the best
 
 ## Join the conversation
 
-We have a channel on the [bitcoin.design](https://bitcoin.design/) Slack community [#lightning-browser-extension](https://bitcoindesign.slack.com/archives/C02591ADXM2) and a [Telegram group](https://t.me/getAlby). Come and join us!
+We have a channel on the [bitcoin.design](https://bitcoin.design/) Discord community [#alby](https://discord.com/channels/903125802726596648/956366795265159189) and a [Telegram group](https://t.me/getAlby). Come and join us!
```

Comment:
```suggestion
We have a channel on the [bitcoin.design](https://bitcoin.design/) Discord community [#alby](https://discord.gg/K7aQ5PErht) and a [Telegram group](https://t.me/getAlby). Come and join us!
```

Reply:
The channel link only works if you already joined the server, otherwise you see this: 

![image](https://github.com/getAlby/lightning-browser-extension/assets/100827540/80bd3c7a-0e30-41c8-9fc3-d0f31a8be867)